analyses/molecular-subtyping-LGAT/01-subset-files-for-LGAT.Rmd

```diff
@@ -313,13 +322,47 @@ lgat_subtypes
 
 ```
 
+Save to subset folder
+
 ```{r}
 
-# save to subset folder
 write_tsv(lgat_subtypes,file.path(subset_dir, "LGAT_snv_subset.tsv"))
 
 ```
 
+### High confidence methylation classification subtypes
+
+```{r}
+lgat_methyl_subtypes <- clinical %>% 
+  dplyr::filter(experimental_strategy == "Methylation", 
+                sample_type == "Tumor", 
+                composition == "Solid Tissue",
+                cohort != "TARGET",
+                dkfz_v12_methylation_subclass_score >= 0.8) %>% 
+  dplyr::mutate(molecular_subtype_methyl = 
+                  case_when(grepl("LGG_MYB", dkfz_v12_methylation_subclass) ~ "LGG, MYB/MYBL1",
+                            dkfz_v12_methylation_subclass == "SEGA" ~ "SEGA, wildtype",
+                            dkfz_v12_methylation_subclass == "A_IDH_LG" ~ "LGG, IDH",
+                            dkfz_v12_methylation_subclass == "PA_INF_FGFR" ~ "LGG, FGFR",
+                            dkfz_v12_methylation_subclass == "AG_MYB" ~ "LGG, MYB/MYBL1",
+                            dkfz_v12_methylation_subclass == "GG" ~ "GNG, wildtype")) %>% 
+  dplyr::select(Kids_First_Biospecimen_ID, Kids_First_Participant_ID, sample_id, molecular_subtype_methyl) %>%
```

Comment:
```suggestion
                            dkfz_v12_methylation_subclass == "SEGA" ~ "SEGA",
                            dkfz_v12_methylation_subclass == "A_IDH_LG" ~ "LGG, IDH",
                            dkfz_v12_methylation_subclass == "PA_INF_FGFR" ~ "LGG, FGFR",
                            dkfz_v12_methylation_subclass == "AG_MYB" ~ "LGG, MYB/MYBL1",
                            dkfz_v12_methylation_subclass == "GG" ~ "GNG")) %>% 
  dplyr::select(Kids_First_Biospecimen_ID, Kids_First_Participant_ID, sample_id, molecular_subtype_methyl) %>%
```

Reply:
Committed suggested change

> Hi @ewafula can you check out what's going on with subtypes including FGFR, I see a few with duplicated FGFR: <img alt="Screenshot 2023-03-03 at 3 33 32 PM" width="489" src="https://user-images.githubusercontent.com/207034/222823625-fcfe7626-d772-491b-8755-76874f50491c.png">

@jharenza, now. It was a bug in the original code. Needed unique() to remove redundancy before applying toString().
https://github.com/PediatricOpenTargets/OpenPedCan-analysis/blob/5fe7544ab3a95a9858d374c0c4504ab269993037/analyses/molecular-subtyping-LGAT/04-LGAT-compile-subtypes.Rmd#L303